docs/sources/write/style-guide/style-conventions/index.md

```diff
@@ -219,3 +219,25 @@ On the published page, this warning renders as follows:
 By disabling authentication requirements, anyone can access your Grafana instance.
 There is a considerable security risk associated with this.
 {{% /admonition %}}
+
+## Semantic line breaks
+
+The [Semantic Line Breaks organization](https://sembr.org/) suggests adding line breaks in your writing. We find this can help with reviews.
```

Comment:
```suggestion
The [Semantic Line Breaks organization](https://sembr.org/) suggests adding line breaks in your writing. 
We find this can help with reviews.
```